rpcs3/Emu/GDB.cpp

```diff
@@ -79,34 +77,33 @@ bool check_errno_again()
 }
 
 std::string u32_to_hex(u32 i) {
-	return fmt::format("%" HEX_U32, i);
+	return fmt::format("%" PRIx32, i);
```

Comment:
Simply use "%x" at all calls, fmt::format is a custom function it's not printf which requires size determinators.